From 279a51d009bc9c2c0be4974d28f7303e72dda35f Mon Sep 17 00:00:00 2001
From: Marc Wimmer <marc@itewimmer.de>
Date: Tue, 3 May 2022 10:24:02 +0200
Subject: [PATCH] for patch

---
 odoo/sql_db.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/odoo/sql_db.py b/odoo/sql_db.py
index 39539612..a546bee0 100644
--- a/odoo/sql_db.py
+++ b/odoo/sql_db.py
@@ -701,7 +701,7 @@ class Connection(object):
         self.dsn = dsn
         self.__pool = pool
 
-    def cursor(self, serialized=True):
+    def cursor(self, serialized=False):
         cursor_type = serialized and 'serialized ' or ''
         _logger.debug('create %scursor to %r', cursor_type, self.dsn)
         return Cursor(self.__pool, self.dbname, self.dsn, serialized=serialized)
-- 
2.34.1

