From eaee4c0fee81093d8e4f933e3ee245909c4ef120 Mon Sep 17 00:00:00 2001
From: Marc Wimmer <marc@itewimmer.de>
Date: Mon, 2 May 2022 09:35:33 +0200
Subject: [PATCH] for patch

---
 odoo/sql_db.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/odoo/sql_db.py b/odoo/sql_db.py
index d7b5630c..5d4d3d9a 100644
--- a/odoo/sql_db.py
+++ b/odoo/sql_db.py
@@ -20,7 +20,7 @@ from decorator import decorator
 import psycopg2
 import psycopg2.extras
 import psycopg2.extensions
-from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT, ISOLATION_LEVEL_READ_COMMITTED, ISOLATION_LEVEL_REPEATABLE_READ
+from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT, ISOLATION_LEVEL_READ_UNCOMMITTED, ISOLATION_LEVEL_READ_COMMITTED, ISOLATION_LEVEL_REPEATABLE_READ
 from psycopg2.pool import PoolError
 from werkzeug import urls
 
@@ -434,7 +434,7 @@ class Cursor(BaseCursor):
             isolation_level = \
                 ISOLATION_LEVEL_REPEATABLE_READ \
                 if self._serialized \
-                else ISOLATION_LEVEL_READ_COMMITTED
+                else ISOLATION_LEVEL_READ_UNCOMMITTED
         self._cnx.set_isolation_level(isolation_level)
 
     @check
-- 
2.34.1

